SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -562,6 +562,20 @@ - (void)handleProtocolEndServiceNAKMessage:(SDLProtocolMessage *)endServiceNAK {
     [self sdl_transitionToStoppedState:endServiceNAK.header.serviceType];
 }
 
+- (void)handleProtocolEndService:(SDLProtocolMessage *)endService forSession:(Byte)session {
+    // Respond with EndService ACK to address SYNC bug
+    SDLProtocol *sdlProtocol = [[SDLProtocol alloc] init];
+    SDLProtocolHeader *header = [SDLProtocolHeader headerForVersion:(UInt8)[SDLGlobals sharedGlobals].protocolVersion.major];
+    header.frameType = SDLFrameTypeControl;
+    header.serviceType = SDLServiceTypeControl;
+    header.frameData = SDLFrameInfoEndServiceACK;
+    header.sessionID = session;
+    SDLProtocolMessage *message = [SDLProtocolMessage messageWithHeader:header andPayload:nil];
+    [sdlProtocol sdl_sendDataToTransport:message.data onService:header.serviceType];
```

Comment:
See my comment on `SDLStreamingAudioLifecycleManager.m`.

Reply:
Fixed in most recent commit.